### ISSUE TITLE
not detect `}finally{`

### ISSUE THREAD

**Issue by saeideng** [cakephp/cakephp-codesniffer#196]
```php
}finally{
} finally {
```
is same for `vendor/bin/phpcs`

**Comment by dereuromark** [comment]
Can you provide a small sniff rule addition for this?

**Comment by dereuromark** [comment]
Is this fixed in current next branch maybe?

**Comment by ADmad** [comment]
Yes the missing space around `finally` are reported by `Squiz.ControlStructures.ControlSignature.SpaceAfterCloseBrace` and `Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword` on `next` branch.
